### ISSUE TITLE
[BUG] Make '@angular/elements' dependency optional

### ISSUE THREAD

**Issue by travist** [formio/angular-formio#398]
## Environment
Please provide as many details as you can:

- Hosting type
  - [x] Form.io
  - [ ] Local deployment
    - Version: 
- Formio.js version:
- Frontend framework:
- Browser:
- Browser version:

## Steps to Reproduce
1.  Install a new angular application
2.  Import this library
3.  Try to compile and you will see the following error "Cannot find module '@angular/elements'."

## Description
I understand how to resolve this problem.... the issue is that all of our current documentation does not require that the user install this as a dependency. I believe this should only be required if they are wanting to create custom components.

Basically what I am saying is that It should allow you to just install the base library to render a form without having to include the @angular/elements dependency.

**Comment by travist** [comment]
@merobal I believe you may have some insights on how to resolve this one.
